src/content/content_manager.cc

```diff
@@ -81,6 +81,12 @@ ContentManager::ContentManager(const std::shared_ptr<Context>& context,
     , session_manager(context->getSessionManager())
     , context(context)
     , timer(std::move(timer))
+#ifdef HAVE_JS
```

Comment:
Why?

Reply:
It's preferred to initialize in the list instead of the body.